packages/remirror__extension-react-tables/src/utils/dom.ts

```diff
@@ -10,17 +12,16 @@ export function h<T extends keyof HTMLElementTagNameMap>(
 
   if (attrs) {
     for (let [key, value] of Object.entries(attrs)) {
-      if (value == null) {
+      if (isNullOrUndefined(value)) {
         continue;
       }
 
       key = key.toLowerCase();
 
-      if (key.length >= 3 && key.startsWith('on') && typeof value === 'function') {
+      if (/^on[a-z]+/.test(key) && typeof value === 'function') {
         element.addEventListener(key.slice(2), value as EventHandler);
       } else if (['class', 'classname'].includes(key)) {
-        const classes = String(Array.isArray(value) ? value.join(' ') : value);
-        element.classList.add(...classes.split(/\s+/));
+        element.classList.add(...cx(value as string).split(' '));
```

Comment:
Rather than spread and split, you could

```suggestion
        element.className= cx(value as string);
```